wlog-zap/internal/provider.go

```diff
@@ -33,7 +32,6 @@ type loggerProvider struct{}
 
 func (lp *loggerProvider) NewLogger(w io.Writer) wlog.Logger {
 	logger, atomicLevel := newZapLogger(w, wlog.InfoLevel, zapcore.EncoderConfig{
-		TimeKey:        wlog.TimeKey,
```

Comment:
Would prefer that this refactor be done as a separate commit that doesn't add wrapped logger

Reply:
Done, opened https://github.com/palantir/witchcraft-go-logging/pull/123/files